Data Structures/Array Implementation/ArrayImplementation(Binary Search).cpp

```diff
@@ -0,0 +1,59 @@
+//A programme for basic Array Implmentation - Binary Search. 
+
+#include <stdio.h>
+
+int binarysearch(int value, int left, int right, int array[]) {
```

Comment:
Function should be named **binarySearch** as per naming conventions. Please adhere to the [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards). 
Make appropriate changes wherever required.